src/ScraperBot/Storage/Plugin/Type/storage.plugin.type.yaml

```diff
@@ -0,0 +1,4 @@
+type: storage
+name: ScraperBot Storage Plugin Type
```

Comment:
GlitcherBot Storage Plugin Type ?

Reply:
Just as per previous comment, really.  Aiming for consistency, and will go about adjusting namespaces as a separate task.

Reply:
agreed